pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -420,6 +432,8 @@ pub mod pallet {
 		/// Process an incoming message.
 		#[pallet::weight(T::WeightInfo::process_msg())]
 		#[pallet::call_index(5)]
+		// TODO: can we modify the name to receive_message() ?
+		// TODO: benchmark me again with max batch limit message
 		pub fn process_msg(
```

Comment:
Currently when receiving a message, you call `process_msg()` and for sending it `process_message()` which is quite confusing. Do you agree to renaming those to `receive_message()` and `send_message()`?
@wischli @mustermeiszer @cdamian

Note that this would imply a breaking change due the extrinsic name is modified

Reply:
No strong opinion. No external integrations, as the extrinsic anyways can only be called by a special origin. So fine for me.